doc/man1/openssl-pkeyutl.pod.in

```diff
@@ -147,6 +155,28 @@ Decrypt the input data using a private key.
 
 Derive a shared secret using the peer key.
 
+=item B<-encap>
+
+Encapsulate the secret using a private key. The result (binary data) is written
+to standard output by default, or else to the file specified with I<-out>. The
+I<-secret> option must also be provided to specify the secret value generated
+in the encapsulation process.
```

Comment:
It seems that the secret value is also *generated* as part of the encapsulation operation (i.e. not provided, but is additional output), then the description can be more clear:
```suggestion
Encapsulate a generated secret using a private key.
The encapsulated result (binary data) is written
to standard output by default, or else to the file specified with I<-out>.
The I<-secret> option must also be provided to specify the output file for the secret value generated
in the encapsulation process.
```
(Fold long lines as necessary). Per `mandoc` specs, start each sentence on a new line.

Reply:
Thanks, accepted